app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -99,34 +99,13 @@ class BindableAdapter<T : Any> internal constructor(
    * The base builder for [BindableAdapter]. This class should not be used directly--use either
    * [SingleTypeBuilder] or [MultiTypeBuilder] instead.
    */
-  abstract class BaseBuilder<BuilderType : Any> {
+  abstract class BaseBuilder(fragment: Fragment) {
```

Comment:
Can we make this constructor private now?

Reply:
Patch applied

Reply:
I think you might've made ``fragment`` a private val rather than making the constructor private. You should instead do:

```kotlin
abstract class BaseBuilder private constructor(fragment: Fragment) {
```